src/stories/Library/Modals/modal-pause/ModalPause.tsx

```diff
@@ -18,59 +21,69 @@ export const ModalPause: React.FC<ModalPauseProps> = ({
   subtitle,
   textWithLink,
   linkText,
-}) => (
-  <Modal shownModal={showModal} classNames="modal-padding modal-pause">
-    <div className="modal-pause__container">
-      <h3 className="text-header-h3">{title}</h3>
-      <div className="mt-48 color-secondary-gray">
-        <p className="text-body-medium-regular">{subtitle}</p>
-      </div>
+}) => {
+  const picker = useRef() as MutableRefObject<Instance>;
 
-      <div className="modal-pause__dropdowns mt-24">
-        <div className="datepickers">
-          <div className="datepicker">
-            <label htmlFor="startDate" className="text-body-medium-regular">
-              Startdato
-            </label>
-            <input
-              type="date"
-              name="startDate"
-              id="startDate"
-              className="datepicker-input"
-              aria-label="startDate"
-            />
-          </div>
+  const calendar = useCallback((node: Node | null) => {
+    const options: Partial<BaseOptions> = {
+      mode: "range",
+      defaultDate: ["2024-01-01", "2024-01-10"],
+      now: "2024-01-19",
+      altInput: true,
+      dateFormat: "d-m-Y",
+      static: true,
+      onReady: (selectedDates, dateStr, instance) => {
+        instance.altInput?.setAttribute("aria-label", "Pause period");
+      },
+    };
+
+    if (node !== null) {
+      picker.current = flatpickr(node, options);
+    }
+  }, []);
+
+  return (
+    <Modal shownModal={showModal} classNames="modal-padding modal-pause">
+      <div className="modal-pause__container">
+        <h3 className="text-header-h3">{title}</h3>
+        <div className="modal-pause__subtitle">
+          <p className="text-body-medium-regular">{subtitle}</p>
+        </div>
 
-          <div className="datepicker">
-            <label htmlFor="startDate" className="text-body-medium-regular">
-              Slutdato
-            </label>
-            <input
-              type="date"
-              name="endDate"
-              id="endDate"
-              className="datepicker-input"
-              aria-label="endDate"
-            />
+        <div className="modal-pause__date-range">
+          <div className="date-range">
+            <div className="date-range__input">
+              <label htmlFor="date-range" className="text-body-medium-regular">
+                Pause period
+              </label>
+              <input
+                id="date-range"
```

Comment:
I have the [same comment as in the other PR](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/896#discussion_r1469376682). 
Would it be possible to make it look a bit more like a field the users can interact with? Using a placeholder text and a little arrow in the right side would do wonders I think.
![image](https://github.com/danskernesdigitalebibliotek/dpl-design-system/assets/28546954/93396090-6203-44b6-968c-89fd92729276)

Reply:
Did it how I did it in dpl-react 👍